client/agenda/AgendaDetailsModal.vue

```diff
@@ -95,19 +95,29 @@ n-modal(v-model:show='modalShown')
             :src='eventDetails.materialsUrl'
             )
         template(v-else-if='state.tab === `slides`')
-          .text-center(v-if='state.isLoading')
-            n-spin(description='Loading slides...')
-          .text-center.p-3(v-else-if='!state.materials || !state.materials.slides || state.materials.slides.length < 1')
-            span No slides submitted for this session.
-          .list-group(v-else)
-            a.list-group-item(
-              v-for='slide of state.materials.slides'
-              :key='slide.id'
-              :href='slide.url'
-              target='_blank'
-              )
-              i.bi.me-2(:class='`bi-filetype-` + slide.ext')
-              span {{slide.title}}
+          n-card(
+            :bordered='false'
+            size='small'
+            )
+            .text-center(v-if='state.isLoading')
+              n-spin(description='Loading slides...')
+            .text-center.p-3(v-else-if='!state.materials || !state.materials.slides || !state.materials.slides.decks || state.materials.slides.decks?.length < 1')
```

Comment:
optional chaining operator `?.` not necessary here as you already validated that `.decks` exists in the previous condition

Reply:
Oops! That was left over from a check whether I could get away without the explicit check that `.decks` is present.